h/views/api/index.py

```diff
@@ -24,4 +24,20 @@ def index(context, request):
         request.route_url, params=["id", "pubid", "user", "userid", "username"]
     )
 
-    return {"links": link_helpers.format_nested_links(api_links, "v1", templater)}
+    return {"links": link_helpers.format_nested_links(api_links, templater)}
+
+
+@api_config(versions=["v2"], route_name="api.index", link_name="index")
```

Comment:
This view function is mostly to illustrate how to respond differently for different versions (and proves that such a thing works.